src/cljs/athens/db.cljs

```diff
@@ -191,14 +191,12 @@
                :current-route       nil
                :loading             true
                :errors              {}
-               :athena              false
+               :athena/open         false
+               :athena/get-recent   '()
```

Comment:
In my last review, I said we should use action verbs for dispatch events. For db keys, we should probably use nouns. Change this to `:athena/recent-items`.